app/src/main/java/org/mozilla/fenix/home/HomeFragment.kt

```diff
@@ -148,6 +149,10 @@ class HomeFragment : Fragment() {
             }
             showRenamedSnackbar()
         }
+
+        override fun onTabsAdded(tabCollection: TabCollection, sessions: List<TabSessionState>) {
```

Comment:
Are there any existing tests for `collecttionStorageObserver` or `onCollectionRenamed`? If so, that might be a good place to test this.

Reply:
Sorry, help needed.
I think we need to create an integration test to verify that `FenixSnackbar.make()` (or `show()` or at least `showTabSavedToCollectionSnackbar()`) gets called after `HomeFragment`'s `collectionStorageObserver.onTabsAdded()` is called.

I've been trying to write something like this in HomeFragmentTest, where `homeFragment` is a spy.

```
        homeFragment.collectionStorageObserver.onTabsAdded(mockk(), mockk())
        verify { homeFragment.showTabSavedToCollectionSnackbar(any(), any(),  any()) }
```
But this never succeeds. Probably because the call is made via `collectionStorageObserver`.
So please help.

Reply:
I see, this is very complicated to write tests for. I think this is good for now 👍

Reply:
Thanks @eliserichards